cmd/mimir/config-descriptor.json

```diff
@@ -13623,10 +13623,10 @@
           "kind": "field",
           "name": "utf8_strict_mode",
           "required": false,
-          "desc": "Enable UTF-8 strict mode. Allows UTF-8 in the matchers for routes and inhibition rules, in silences, and in the labels for alerts. It is recommended to check both alertmanager_matchers_disagree and alertmanager_matchers_incompatible metrics before using this mode as otherwise some tenant configurations might fail to load.",
+          "desc": "Enable UTF-8 strict mode. Allows UTF-8 characters in the matchers for routes and inhibition rules, in silences, and in the labels for alerts. It is recommended to check both alertmanager_matchers_disagree and alertmanager_matchers_incompatible metrics before using this mode as otherwise some tenant configurations might fail to load.",
```

Comment:
This is a real pedantic nit, but it's not just characters we allow like 🙂, but also their representation as hexademical (`U+1F642`) and byte sequences `\xf0\x9f\x99\x82`. I think it's best kept as either just UTF-8 or unicode?

Reply:
I shouldn't have used the word characters there though, unicode doesn't have the notion of characters.